en-US/win10/SetupTPM.md

```diff
@@ -287,7 +287,7 @@ Note that **sTPM is intended for development purposes only and does not provide
         bcdedit /set {current} integrityservices disable
         bcdedit /set testsigning on
 
-4. Copy the matching ACPI table from [here][3] to RPi2 _C:\Windows\System32\ACPITABL.dat_.
+4. Copy the ACPI table from [here][4] to MBM/RPi2 _C:\Windows\System32\ACPITABL.dat_.
```

Comment:
Is step 4 the one that needs to be clarified?
Basically, the user needs to copy the ACPITABL.dat file published on GitHub to the filesystem of the Athens device. The location on the Athens' filesystem (SD card) that this file should go is: C:\Windows\System32